content/docs/user-guide/scripting/script-canvas/programmer-guide/custom-nodes/custom-nodeable-nodes.md

```diff
@@ -0,0 +1,259 @@
+---
+linktitle: Custom Nodeable Nodes
+title: Creating Custom Nodeable Nodes in Script Canvas
+description: Learn how to create custom Script Canvas Nodeable Nodes in Open 3D Engine (O3DE).
+weight: 100
+---
+
+This topic guides you how to create custom Script Canvas Nodeable Nodes step by step.
+
+You'll see the term _nodeable_ used throughout the O3DE source. 
+A nodeable can refer to both the node that appears in the Node Palette as a result of the AzAutoGen processing,
+and the mechanism by which a compiled Script Canvas graph can invoke C++ functions.
+
+## Prerequisite: Adding support for custom nodeable nodes to a Gem
+In your Gem's `CMakeLists.txt`, add a section for `AUTOGEN_RULES` and declare `Gem::ScriptCanvas` as build dependency.
```

Comment:
We should probably make a suggestion about which `CMakeLists.txt`, since there's a hierarchy of them.
```suggestion
In your Gem's `Code/CMakeLists.txt`, add a section for `AUTOGEN_RULES` and declare `Gem::ScriptCanvas` as a build dependency.
```

Reply:
done